x-pack/legacy/plugins/monitoring/server/telemetry_collection/register_monitoring_collection.ts

```diff
@@ -4,18 +4,22 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { telemetryCollectionManager } from '../../../../../../src/legacy/core_plugins/telemetry/server';
-import { getAllStats } from './get_all_stats';
+import { TelemetryCollectionManagerPluginSetup } from 'src/plugins/telemetry_collection_manager/server';
+import { getAllStats, CustomContext } from './get_all_stats';
 import { getClusterUuids } from './get_cluster_uuids';
 import { getLicenses } from './get_licenses';
 
-export function registerMonitoringCollection() {
+export function registerMonitoringCollection(
+  telemetryCollectionManager: TelemetryCollectionManagerPluginSetup,
+  customContext: CustomContext
+) {
   telemetryCollectionManager.setCollection({
     esCluster: 'monitoring',
     title: 'monitoring',
     priority: 2,
```

Comment:
Question: Should we not consider explicitly declaring collection priority in the `telemetryCollectionManager` rather than embedding priorities in here? That way, we could make the priority order more explicit.

Reply:
that sounds like a neat suggestion, but I'd rather tackle it in a new issue since it's not related to the migration, itself